### PR TITLE
First stab at resource manager docs

### PR DESCRIPTION
**Pull request by shuds13**
Documentation for variable resources.

### CODE REVIEW

File: docs/function_guides/allocator.rst [hunk 1]
```diff
@@ -95,22 +95,25 @@ routine can be found in ``libE_info``, passed into the allocation function::
 
     libE_info =  {'exit_criteria': dict,               # Criteria for ending routine
                   'elapsed_time': float,               # Time elapsed since start of routine
-                  'manager_kill_canceled_sims': bool,  # True if manager is canceling simulations
+                  'manager_kill_canceled_sims': bool,  # True if manager is to send kills to cancelled simulations
                   'given_count': int,                  # Total number of points given for simulation function evaluation
                   'returned_count': int,               # Total number of points returned from simulation function evaluations
                   'given_back_count': int,             # Total number of evaluated points given back to a generator function
-                  'sim_max_given': bool}               # True if the maximum number of simulations has been performed
+                  'sim_max_given': bool}               # True if the `sim_max` simulations have been given out to workers
```
Comment (jlnav) [inline docs/function_guides/allocator.rst:102]: ~~the~~

File: docs/function_guides/allocator.rst [hunk 2]
```diff
@@ -95,22 +95,25 @@ routine can be found in ``libE_info``, passed into the allocation function::
...
 
-Besides the initial allocation function check, the remaining values above are useful
-for very efficient control over how work is distributed to workers, especially for
-authors of persistent generator functions. Generators that construct models based
+
+In most supplied examples, the allocation function will just return once `sim_max_given`
```
Comment (jlnav) [inline docs/function_guides/allocator.rst:105]: extra ticks around sim_max_given

File: docs/function_guides/allocator.rst [hunk 3]
```diff
@@ -95,22 +95,25 @@ routine can be found in ``libE_info``, passed into the allocation function::
...
+is ``True``, but the user could choose to do something different, such as cancel points or
+keep returning completed points to the generator. Generators that construct models based
 on *all evaluated points*, for example, may need simulation work units at the end
-of a routine to be returned to the generator anyway.
+of an ensemble to be returned to the generator anyway.
+
+Alternatively, users can use `elapsed_time` to track the ensembles runtime inside their
```
Comment (jlnav) [inline docs/function_guides/allocator.rst:111]: extra ticks around elapsed_time

File: docs/platforms/platforms_index.rst [hunk 4]
```diff
@@ -110,69 +110,17 @@ the :doc:`examples<example_scripts>`.
...
-    :scale: 40
-    :align: center
-
-This can be resolved within the Executor definition in the calling script. Set the
-parameter ``zero_resource_workers`` to a list of worker IDs that shouldn't have
-system resources assigned. For example, when using a single instance of Persistent
-:doc:`APOSMM<../examples/aposmm>` as your ``gen_f``, the Executor definition
-may resemble::
+are still automatically assigned system resources. This can be resolved by
+using using :ref:`zero resource workers<zero_resource_workers>`.
```
Comment (jlnav) [inline docs/platforms/platforms_index.rst:122]: ~~using~~

File: docs/resource_manager/overview.rst [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+To set CUDA_VISIBLE_DEVICES to slots in this case, use the  ``multiplier`` argument in the ``set_env_to_slots`` function:
+
+.. code-block:: python
+    :emphasize-lines: 2
+
+    resources = Resources.resources.worker_resources
+    resources.set_env_to_slots("CUDA_VISIBLE_DEVICES", multiplier=2)
+
+
+Setting more resource sets that workers
```
Comment (jlnav) [inline docs/resource_manager/overview.rst:140]: than workers

File: docs/resource_manager/overview.rst [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+
+    libE_specs['num_resource_sets'] = 8
+
+and running on 5 workers.
+
+Also, this can be set on the command line as a convenience.
+
+.. code-block:: bash
+
+    python run_ensemble.py --ncomms local --nworkers 5 --nresource_sets 8
```
Comment (jlnav) [inline docs/resource_manager/overview.rst:161]: --comms, not --ncomms

**Review by jlnav — APPROVED** [review]

File: docs/function_guides/allocator.rst (hunk 2 above)
Comment (jmlarson1) [reply docs/function_guides/allocator.rst:105]: Don't we usually have ticks in such a place?

File: docs/function_guides/allocator.rst (hunk 3 above)
Comment (jmlarson1) [reply docs/function_guides/allocator.rst:111]: Don't we usually have ticks in such a place?

**Comment by shuds13** [comment]
@jlnav Great catches!

File: docs/function_guides/allocator.rst (hunk 2 above)
Comment (jlnav) [reply docs/function_guides/allocator.rst]: as in, extra ticks needed in rst, I think? ``sim_max_given`` vs `sim_max_given`. Should've been clearer!

(looks like on github both render the same...)

**Pull request merged**